### ISSUE TITLE
Does not work, period, on Win 10 Home latest build

### ISSUE THREAD

**Issue by tzackin** [stefankueng/tools#39]
I scanned the registry for all references to the dll but is there an easier way to uninstall this since it is a complete no-op?  Thanks.

**Comment by stefankueng** [comment]
which tool? and what exactly does not work?

**Comment by tzackin** [comment]
StExBar.  Does it work in Win 10 Home?  If not, is there an easy way to remove it?  Thanks.

**Comment by stefankueng** [comment]
yes, it works fine on Win10 home. And you can uninstall it the same way any other installed app.

**Comment by tzackin** [comment]
Thanks.  I guess I didn't see it in the installed programs list.  BTW, as I said it didn't  seem to work although, from the documentation, I really couldn't figure out how to display your tool bar in Explorer.  If it is supposed to appear automatically once installed then there is something wrong.  Perhaps Norton Security or Malwarebytes disabled something?

**Comment by stefankueng** [comment]
in the "view" tab of explorer, the very right button "options". If you click on it there's an entry "StExBar", click on that.
But since you're using Norton, you should be used to problem :-)

Also, if third-party plugins are disabled then StExBar can't show up. Type "internet options" in the start menu, then in the options dialog go to the "Extended" tab. Somewhere there's an option "enable browser plugins from third parties" (or something like that, don't have an English OS here). That has to be enabled.

**Comment by tzackin** [comment]
Under **Internet Properties** | **Advanced** there is the option "Enable third-party browser extensions" which is checked so that is not the problem.  I am assuming that is the option you are referring to.

**Comment by stefankueng** [comment]
yes, that's what I'm referring to. And it still doesn't show up in the ribbon?
![image](https://user-images.githubusercontent.com/665591/88433544-972e4280-cdfe-11ea-8df8-b8fb94b1b378.png)

**Comment by tomhundt** [comment]
Try running Internet Explorer, go to Tools (gear icon) > Manage Add-ons > Toolbars and Extensions, and see if there is an entry for StExBar there, set to Disabled or New, which needs to be Enabled (button lower right).
